docs/sources/next/javascript-api/k6-experimental/browser/frame/ishidden.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 'isHidden(selector[, options])'
+excerpt: 'Browser module: frame.isHidden(selector[, options]) method'
+---
+
+# isHidden(selector[, options])
+
+{{% admonition type="warning" %}}
+
+Use locator-based [`locator.isHidden([options])`](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/locator/ishidden/) instead.
```

Comment:
Just out of curiosity, is there a specific reason why we recommend using the locator-based method?

Reply:
`Locators` are seen to be better since they will auto wait and retry the search for the element with the given `selector`. This means that we can set a locator, navigate to a page, and then still work with the locator after the navigation occurs (as long as the element is actually on the new page). Whereas with the `frame` APIs, once a navigation occurs, we cannot use the old reference to the element that was setup before the navigation. In a nutshell, `locators` are safer.

Reply:
Ah, that does seem better, thanks for the explanation! 🙇

Reply:
We explain locators with a Page Object Model example [here](https://k6.io/docs/using-k6-browser/recommended-practices/page-object-model/). We might expand on it.